### ISSUE TITLE
the model is pre-trained on ImageNet？

### ISSUE THREAD

**Issue by wuhao9714** [superhans/colorfromlanguage#4]
Hello, you mentioned in your paper that the model is pre-trained on ImageNet.Do I need additional training on ImageNet before following your instructions?Alternatively, Resnet101 generates image_features. H5, which represents the process of training the model on ImageNet.

**Comment by superhans** [comment]
It has been some time since I performed this work. 

But the critical step was pre-training the model on ImageNet. The model we trained was VGG based. We took the model from here : `https://github.com/richzhang/colorization` (I remember them to be VGG based, but in the current repo, they appear to be AlexNet based). 

We converted all the caffe weights to pytorch weights and initialized the first few layers with these weights. In some cases, It was not size compatible with our model, so we did an interpolation of the weight matrices. 

In short, it is critical that you pre-train the model on the colorization task on ImageNet (although we took a shortcut by using the pretrained model from Colorful Image Colorization paper). I still probably have the model on some hard-drive, so I can look it up (no guarantees).

**Comment by wuhao9714** [comment]
Thank you very much for taking the time to answer my questions.I would be very grateful if you could find this pre-trained model.
